lineax/_solver/tridiagonal.py

```diff
@@ -12,11 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from functools import partial
 from typing import Any
 from typing_extensions import TypeAlias
 
 import jax.lax as lax
 import jax.numpy as jnp
+from jax._src.interpreters import mlir
```

Comment:
Can we do `import jax.interpreters.mlir as mlir` instead? (This is both a style comment, and a question on whether the public API is enough?)